cli/ballerina-cli/src/main/java/io/ballerina/cli/cmd/RunCommand.java

```diff
@@ -182,7 +195,8 @@ public void printLongDesc(StringBuilder out) {
 
     @Override
     public void printUsage(StringBuilder out) {
-        out.append("  bal run [--offline] [<balfile> | <project-path> | executable-jar]\n" +
+        out.append("  bal run [--debug] <executable-jar>\n");
```

Comment:
Shouldn't we mention that its mandatory to specify the debug port, in the output?

Reply:
```suggestion
        out.append("  bal run [--debug <port>] <executable-jar>\n");
```